docs/Dockerfile

```
@@ -0,0 +1,89 @@
+# .git) must be present before bundle install resolves it.
+COPY . /app/
+
+# Frozen off: the path-gem's git-ls-files gemspec digest can't be pinned across
+# commits; the lockfile still pins every other gem.
+RUN bundle config set frozen false && bundle install && \
+    rm -rf ~/.bundle/ "${BUNDLE_PATH}"/ruby/*/cache
+
+# JS deps (Tailwind CLI + daisyUI), then assets:precompile — enhanced by the
+# css:build rake task so it runs `bun run build:css` before propshaft fingerprints.
+RUN bun install --frozen-lockfile
+RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)